ouroboros-network/test/Test/Ouroboros/Network/Testnet/Simulation/Node.hs

```diff
@@ -910,9 +917,13 @@ diffusionSimulation
               , NodeKernel.iNtcBearer         = makeFDBearer
               , NodeKernel.iRng               = rng
               , NodeKernel.iDomainMap         = dMapVar
-              , NodeKernel.iLedgerPeersConsensusInterface
-                                        = LedgerPeersConsensusInterface
-                                        $ \_ -> return Nothing
+              , NodeKernel.iLedgerPeersConsensusInterface =
+                  LedgerPeersConsensusInterface $
+                    \_ -> return
+                        $ Just
+                        $ Map.elems
+                        $ accPoolStake
+                        $ getLedgerPools ledgerPeers
```

Comment:
These are static ledger pools, it wouldn't be difficult to have a script which makes them changing over time.  Please add an issue, so we'll do that in the future.

Reply:
Done!